### ISSUE TITLE
Investigate issues on Node 18.7+

### ISSUE THREAD

**Issue by Comandeer** [Comandeer/esm-loader-manager#5]
See https://github.com/Comandeer/esm-loader-manager/runs/7542860761?check_suite_focus=true

**Comment by Comandeer** [comment]
Error:

```
Error [ERR_LOADER_CHAIN_INCOMPLETE]: "file:///home/comandeer/repos/tools/esm-loader-manager/src/index.js 'load'" did not call the next hook in its chain and did not explicitly signal a short circuit. If this is intentional, include `shortCircuit: true` in the hook's return.
    at new NodeError (node:internal/errors:387:5)
    at ESMLoader.load (node:internal/modules/esm/loader:624:13)
    at async ESMLoader.moduleProvider (node:internal/modules/esm/loader:472:11) {
  code: 'ERR_LOADER_CHAIN_INCOMPLETE'
}
```

It seems to be connected with loader chaining introduced in Node 18.6.0.

**Comment by Comandeer** [comment]
Adding `shortCircuit: true` option to the loader should do the trick.

**Comment by Comandeer** [comment]
Fixed in 91aa46dc3c4becbb3a2cf31bdb6e021eef0d0fc4.
